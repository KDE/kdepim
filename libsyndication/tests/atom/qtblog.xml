<?xml version="1.0" encoding="utf-8"?>
<!-- 
     - Movable type 3.2 Atom 1.0 feed 
     - xml:base and xml:lang usage
     - 
     --> 
<feed xmlns="http://www.w3.org/2005/Atom">
    <title>A Qt Blog</title>
    <link rel="alternate" type="text/html" href="http://www.blogistan.co.uk/qt/" />
    <link rel="self" type="application/atom+xml" href="http://www.blogistan.co.uk/qt/atom.xml" />
   <id>tag:www.blogistan.co.uk,2006:/qt//1</id>
    <link rel="service.post" type="application/atom+xml" href="http://www.blogistan.co.uk/mt/mt-atom.cgi/weblog/blog_id=1" title="A Qt Blog" />
    <updated>2006-03-16T01:06:35Z</updated>
    <subtitle>The blog where it&apos;s OK to feed the Trolls</subtitle>
    <generator uri="http://www.sixapart.com/movabletype/">Movable Type 3.2</generator>
 
<entry>
    <title>KBarcode 2.0.0 released</title>
    <link rel="alternate" type="text/html" href="http://www.blogistan.co.uk/qt/2006/03/kbarcode_200_released.php" />
    <link rel="service.edit" type="application/atom+xml" href="http://www.blogistan.co.uk/mt/mt-atom.cgi/weblog/blog_id=1/entry_id=2692" title="KBarcode 2.0.0 released" />
    <id>tag:www.blogistan.co.uk,2006:/qt//1.2692</id>
    
    <published>2006-03-16T01:06:27Z</published>
    <updated>2006-03-16T01:06:35Z</updated>
    
    <summary>From KDE Apps: Version 2.0.0 of KBarcode, a KDE application for barcode and label printing, has been released. New features include printing from data sources including SQL queries, CSV files and the KDE address book, JavaScript support, new barcode formats,...</summary>
    <author>
        <name>Indigo Jo</name>
        <uri>http://www.blogistan.co.uk/blog/</uri>
    </author>
            <category term="KDE Apps" />
    
    <content type="html" xml:lang="en" xml:base="http://www.blogistan.co.uk/qt/">
        <![CDATA[<p><a title="ï¿½KBarcode KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=9924">From KDE Apps</a>:</p>

<p>Version 2.0.0 of <a href="http://www.kbarcode.net/">KBarcode</a>, a KDE application for barcode and label printing, has been released.  New features include printing from data sources including SQL queries, CSV files and the KDE address book, JavaScript support, new barcode formats, batch printing configured through a wizard, and a rewritten PDF manual.  Downloads (source, SUSE 10 and FC3 RPM) <a href="http://www.kbarcode.net/Stable.6.0.html">here</a>, screenshots <a href="http://www.kbarcode.net/Screenshots.29.0.html">here</a>.  Requires KDE 3.3.</p>
]]>
        

    </content>
</entry>
<entry>
    <title>Akademy 2006 will be in Dublin</title>
    <link rel="alternate" type="text/html" href="http://www.blogistan.co.uk/qt/2006/03/akademy_2006_will_be_in_dublin.php" />
    <link rel="service.edit" type="application/atom+xml" href="http://www.blogistan.co.uk/mt/mt-atom.cgi/weblog/blog_id=1/entry_id=2691" title="Akademy 2006 will be in Dublin" />
    <id>tag:www.blogistan.co.uk,2006:/qt//1.2691</id>
    
    <published>2006-03-15T23:13:56Z</published>
    <updated>2006-03-15T23:14:02Z</updated>
    
    <summary>From KDE Apps: The annual KDE world summit, Akademy, will be held at Trinity College in Dublin, Republic of Ireland, from 23rd to 30th September. The event will consist of a contributors&apos; conference, the KDE e.V. AGM, and a five-day...</summary>
    <author>
        <name>Indigo Jo</name>
        <uri>http://www.blogistan.co.uk/blog/</uri>
    </author>
            <category term="Events" />
            <category term="KDE Community" />
    
    <content type="html" xml:lang="en" xml:base="http://www.blogistan.co.uk/qt/">
        <![CDATA[<p><a title="Akademy 2006 is in Dublin this September" href="http://dot.kde.org/1142439906/">From KDE Apps</a>:</p>

<p>The annual KDE world summit, Akademy, will be held at <a href="http://www.tcd.ie/">Trinity College</a> in Dublin, Republic of Ireland, from 23rd to 30th September.  The event will consist of a contributors' conference, the <a href="http://ev.kde.org/">KDE e.V.</a> AGM, and a five-day hacking session with workshops (26th-30th).  The conference will be organised by Marcus Furlong who has organised a GUADEC in Dublin in the past and "contributes to KDE by taking a sh*tload of work on him".</p>
]]>
        

    </content>
</entry>
</feed> 

