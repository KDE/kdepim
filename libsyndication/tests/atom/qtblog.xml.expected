# Feed begin ######################
title: #A Qt Blog#
link: #http://www.blogistan.co.uk/qt/#
description: #The blog where it&apos;s OK to feed the Trolls#
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2692#
title: #KBarcode 2.0.0 released#
link: #http://www.blogistan.co.uk/qt/2006/03/kbarcode_200_released.php#
description: #From KDE Apps: Version 2.0.0 of KBarcode, a KDE application for barcode and label printing, has been released. New features include printing from data sources including SQL queries, CSV files and the KDE address book, JavaScript support, new barcode formats,...#
content: #<p><a title="�KBarcode KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=9924">From KDE Apps</a>:</p> <p>Version 2.0.0 of <a href="http://www.kbarcode.net/">KBarcode</a>, a KDE application for barcode and label printing, has been released. New features include printing from data sources including SQL queries, CSV files and the KDE address book, JavaScript support, new barcode formats, batch printing configured through a wizard, and a rewritten PDF manual. Downloads (source, SUSE 10 and FC3 RPM) <a href="http://www.kbarcode.net/Stable.6.0.html">here</a>, screenshots <a href="http://www.kbarcode.net/Screenshots.29.0.html">here</a>. Requires KDE 3.3.</p>#
datePublished: #Thu Mar 16 02:06:27 2006#
dateUpdated: #Thu Mar 16 02:06:27 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2691#
title: #Akademy 2006 will be in Dublin#
link: #http://www.blogistan.co.uk/qt/2006/03/akademy_2006_will_be_in_dublin.php#
description: #From KDE Apps: The annual KDE world summit, Akademy, will be held at Trinity College in Dublin, Republic of Ireland, from 23rd to 30th September. The event will consist of a contributors&apos; conference, the KDE e.V. AGM, and a five-day...#
content: #<p><a title="Akademy 2006 is in Dublin this September" href="http://dot.kde.org/1142439906/">From KDE Apps</a>:</p> <p>The annual KDE world summit, Akademy, will be held at <a href="http://www.tcd.ie/">Trinity College</a> in Dublin, Republic of Ireland, from 23rd to 30th September. The event will consist of a contributors' conference, the <a href="http://ev.kde.org/">KDE e.V.</a> AGM, and a five-day hacking session with workshops (26th-30th). The conference will be organised by Marcus Furlong who has organised a GUADEC in Dublin in the past and "contributes to KDE by taking a sh*tload of work on him".</p>#
datePublished: #Thu Mar 16 00:13:56 2006#
dateUpdated: #Thu Mar 16 00:13:56 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Events#
# Category end ####################
# Category begin ##################
term: #KDE Community#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2690#
title: #Qt3 and QSA updates#
link: #http://www.blogistan.co.uk/qt/2006/03/qt3_and_qsa_updates.php#
description: #Trolltech has released version 3.3.6 of Qt. This version adds support for Windows Vista and the Mac/Intel architecture, various translations contributed by Novell, and among many bug fixes, a fix for compilation on GCC 4.1 on the X11 platform. The...#
content: #<p>Trolltech has released version 3.3.6 of Qt. This version adds support for Windows Vista and the Mac/Intel architecture, various translations contributed by Novell, and among many bug fixes, a fix for compilation on GCC 4.1 on the X11 platform. The download is available at the Trolltech FTP site <a href="ftp://ftp.trolltech.com/qt/source/">here</a>; as it is Qt 3, there is no free Windows version.</p> <p><a title="Trolltech - Trolltech Releases Qt 3.3.6" href="http://www.trolltech.com/newsroom/announcements/00000246.html">Trolltech - Trolltech Releases Qt 3.3.6</a></p> <p>Trolltech has also released new versions of QSA (<a href="http://www.trolltech.com/products/qsa/index.html">Qt Script for Applications</a>): version 1.2.1 for Qt 4 and version 1.1.4 for Qt 3. Both are mostly bug fix releases; the changelogs can be found here: <a href="http://www.trolltech.com/developer/changes/changes-qsa-1.1.4.html">1.1.4</a> and <a href="http://www.trolltech.com/developer/changes/changes-qsa-1.2.1.html">1.2.1</a>.</p> <p>Announcement: <a href="http://www.trolltech.com/newsroom/announcements/00000247.html">Trolltech Releases QSA 1.1.4 and 1.2.1</a></p>#
datePublished: #Thu Mar 16 00:05:05 2006#
dateUpdated: #Thu Mar 16 00:05:05 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Qt#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2688#
title: #Kexi: second beta release#
link: #http://www.blogistan.co.uk/qt/2006/03/kexi_second_beta_release.php#
description: #From KDE Apps: A second beta of Kexi, the KOffice database, has been released. This features a new CopyCenter script package, much &quot;usability polish&quot; and various stability and data handling fixes. Released as part of the recent KOffice 1.5 beta...#
content: #<p><a title="�Kexi KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=12365">From KDE Apps</a>:</p> <p>A second beta of <a href="http://www.kexi-project.org/">Kexi</a>, the KOffice database, has been released. This features a new CopyCenter script package, much "usability polish" and various stability and data handling fixes. Released as part of the recent KOffice 1.5 beta (<a href="http://koffice.org/announcements/announce-1.5-beta2.php">announcement here</a>; full changelog <a href="http://www.koffice.org/announcements/changelog-1.5beta2.php">here</a>. Binaries currently available for SUSE 9.3 and 10, Kubuntu and Debian Sid; full details at the announcement link above.</p>#
datePublished: #Wed Mar 15 15:26:54 2006#
dateUpdated: #Wed Mar 15 15:26:54 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2686#
title: #Luma 2.3 released#
link: #http://www.blogistan.co.uk/qt/2006/03/luma_23_released.php#
description: #From KDE Apps: Version 2.3 of Luma, a &quot;tool for easy and userfriendly managment of LDAP data and a platform for developing custom LDAP applications&quot;, has been released. The new version has several new features and a substantial number of...#
content: #<p><a title="Luma - LDAP Editor/Application Framework - KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=9771">From KDE Apps</a>:</p> <p>Version 2.3 of <a href="http://luma.sourceforge.net/">Luma</a>, a "tool for easy and userfriendly managment of LDAP data and a platform for developing custom LDAP applications", has been released. The new version has several new features and a substantial number of fixes and improvements - see the KDE Apps page. Screenshots <a href="http://luma.sourceforge.net/screenshots.html">here</a>, source downloads and requirement details <a href="http://luma.sourceforge.net/download.html">here</a>. Written in Python (2.3 or later) and so requires PyQt (3.10 or later). Also requires Qt 3.2.</p>#
datePublished: #Tue Mar 14 19:21:51 2006#
dateUpdated: #Tue Mar 14 19:21:51 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Qt Applications#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2684#
title: #QTM 0.3.1 released#
link: #http://www.blogistan.co.uk/qt/2006/03/qtm_031_released.php#
description: #From KDE Apps: QTM, a Qt 4-based client for Movable Type weblogs (and others supporting the same interfaces) has been released. New features include support for custom RSS excerpts, a redesigned entry settings panel, and some bug fixes. Screenshots are...#
content: #<p><a title="�Catkin QTM KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=34289">From KDE Apps</a>:</p> <p>QTM, a Qt 4-based client for Movable Type weblogs (and others supporting the same interfaces) has been released. New features include support for custom RSS excerpts, a redesigned entry settings panel, and some bug fixes. Screenshots are at KDE Apps; downloads (source only at present) there and at <a href="http://sourceforge.net/project/showfiles.php?group_id=112622&amp;package_id=177004&amp;release_id=401532">SourceForge</a>.</p>#
datePublished: #Tue Mar 14 14:37:54 2006#
dateUpdated: #Tue Mar 14 14:37:54 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Qt4 Applications#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2681#
title: #Protoeditor 0.7 released#
link: #http://www.blogistan.co.uk/qt/2006/03/protoeditor_07_released.php#
description: #From KDE Apps: Protoeditor is back in development after an 8-month hiatus, and version 0.7 has been released. It is a KDE-based editor for debugging scripts (currently only PHP) interactively using the free DBG debugger or Xdebug (see this page...#
content: #<p><a title="�Protoeditor KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=20567">From KDE Apps</a>:</p> <p><a href="http://protoeditor.sourceforge.net/">Protoeditor</a> is back in development after an 8-month hiatus, and version 0.7 has been released. It is a KDE-based editor for debugging scripts (currently only PHP) interactively using the free DBG debugger or Xdebug (see <a href="http://protoeditor.sourceforge.net/index.php?page=about">this page</a> for details). This version now allows executing and debugging local scripts, improves Xdebug support, and fixes various bugs. Screenshots <a href="https://sourceforge.net/project/screenshots.php?group_id=129439">here</a>, downloads (source only) <a href="http://sourceforge.net/project/showfiles.php?group_id=129439&amp;package_id=141889&amp;release_id=401317">here</a>.</p>#
datePublished: #Tue Mar 14 11:17:03 2006#
dateUpdated: #Tue Mar 14 11:17:03 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2679#
title: #B &amp; S book expected this June#
link: #http://www.blogistan.co.uk/qt/2006/03/b_s_book_expected_this_june.php#
description: #Jasmin Blanchette blogs that the Qt 4 version of his and Mark Summerfield&apos;s well-known Qt 3 book is due for release this June and that the final manuscript has been submitted to the publishers. It includes three new chapters and...#
content: #<p>Jasmin Blanchette blogs that the Qt 4 version of <a href="http://www.amazon.co.uk/exec/obidos/ASIN/0131240722/qid=1142250668/sr=8-1/ref=pd_ka_1/026-8337942-1596445">his and Mark Summerfield's well-known Qt 3 book</a> is due for release this June and that the final manuscript has been submitted to the publishers. It includes three new chapters and an appendix, on item view classes, writing plugins, embedded programming, and introducing C++ to Java and C# programmers. The book has been updated for Qt 4.1, not just Qt 4, and concentrates on explaining how to program rather than "rehashing or summarizing Qt's extensive online documentation".</p> <p><a title="QDevBlog: Qt 4 book expected in June 2006" href="http://blogs.qtdeveloper.net/archives/2006/03/01/qt-4-book-expected-in-june-2006/">QDevBlog: Qt 4 book expected in June 2006</a></p>#
datePublished: #Mon Mar 13 12:55:06 2006#
dateUpdated: #Mon Mar 13 12:55:06 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Books#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2678#
title: #Office packages updated#
link: #http://www.blogistan.co.uk/qt/2006/03/office_packages_updated.php#
description: #From KDE Apps ([1], [2]): A second beta of KOffice has been released, featuring a new charting engine donated by Klarälfdalens Datakonsult (Matthias Kalle Dalheimer&apos;s company) and &quot;an initial implementation&quot; of the OpenDocument format. Announcement, with more details on major...#
content: #<p>From KDE Apps (<a href="http://dot.kde.org/1142179297/">[1]</a>, <a href="http://dot.kde.org/1142077964/">[2]</a>):</p> <p>A second beta of KOffice has been released, featuring a new charting engine donated by <a href="http://http://www.klaralvdalens-datakonsult.se/">Klar&auml;lfdalens Datakonsult</a> (Matthias Kalle Dalheimer's company) and "an initial implementation" of the OpenDocument format. Announcement, with more details on major improvements, <a href="http://www.koffice.org/announcements/announce-1.5-beta2.php">here</a>, full changelog <a href="http://www.koffice.org/announcements/changelog-1.5beta2.php">here</a>, download links <a href="http://download.kde.org/download.php?url=unstable/koffice-1.5-beta2">here</a> (there are also binaries for SUSE Linux). The final release is <a href="http://developer.kde.org/development-versions/koffice-1.5-release-plan.html">due in April</a> (contrary to the K.N announcement).</p> <p>Also, <a href="http://www.openoffice.org/">OpenOffice.org</a> 2.0.2 has been released, featuring the <a href="http://opensource.bureau-cornavin.com/kab/">KDE AddressBook Connector</a> and an icon set based on KDE's Crystal icons (not yet complete; the announcement points anyone who might want to help to the <a href="http://artax.karlin.mff.cuni.cz/~kendy/ooo/icons/status/">status page</a>, which loads the entire icon set into the browser, so be warned).</p>#
datePublished: #Mon Mar 13 12:29:21 2006#
dateUpdated: #Mon Mar 13 12:29:21 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2677#
title: #KPoGre v1.3.7 released#
link: #http://www.blogistan.co.uk/qt/2006/03/kpogre_v137_released.php#
description: #From KDE Apps: Version 1.3.7 of KPoGre, a PostgreSQL administrator for KDE, has been released. Changelog can be found on the KDE Apps page; screenshots here, downloads at Sourceforge here. Requires KDE 3.5....#
content: #<p><a title="�KPoGre KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=12943">From KDE Apps</a>:</p> <p>Version 1.3.7 of <a href="http://kpogre.sourceforge.net/">KPoGre</a>, a PostgreSQL administrator for KDE, has been released. Changelog can be found on the KDE Apps page; screenshots <a href="http://kpogre.sourceforge.net/screenshots/screenshots.html">here</a>, downloads at Sourceforge <a href="http://sourceforge.net/project/showfiles.php?group_id=38943">here</a>. Requires KDE 3.5.</p>#
datePublished: #Mon Mar 13 12:27:59 2006#
dateUpdated: #Mon Mar 13 12:27:59 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2676#
title: #QSClient 1.2 release candidate released#
link: #http://www.blogistan.co.uk/qt/2006/03/qsclient_12_release_candidate.php#
description: #From KDE Apps: A third release candidate of QSClient (English page here), a Qt-based general internet client which currently supports 27 different types of internet service, has been released. This version can be installed on a USB stick and features...#
content: #<p><a title="�QSClient KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=29637">From KDE Apps</a>:</p> <p>A third release candidate of <a href="http://www.fenner.info/">QSClient</a> (English page <a href="http://www.fenner.info/clients/docs/index.html">here</a>), a Qt-based general internet client which currently supports 27 different types of internet service, has been released. This version can be installed on a USB stick and features a SSH agent to activate keys on Windows and Linux; there is also improved documentation and various bug-fixes (see changelog on KDE Apps page). Screenshots and downloads (Linux and Windows installers, Windows source) at KDE Apps.</p>#
datePublished: #Mon Mar 13 12:21:44 2006#
dateUpdated: #Mon Mar 13 12:21:44 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Qt Applications#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2675#
title: #Kopete beta released#
link: #http://www.blogistan.co.uk/qt/2006/03/kopete_be.php#
description: #From KDE Apps: A beta of version 0.12 of Kopete, the KDE multi-protocol chat client, has been released. The new version features support for Adium window themes as opposed to the old XSL-based ones, a new Yahoo protocol implementation which...#
content: #<p><a title="�Kopete KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=9891&amp;PHPSESSID=d324daeb8900fa39aaa0cd5e22cc5b8f">From KDE Apps</a>:</p> <p>A beta of version 0.12 of <a href="http://kopete.kde.org/">Kopete</a>, the KDE multi-protocol chat client, has been released. The new version features support for Adium window themes as opposed to the old XSL-based ones, a new Yahoo protocol implementation which is also in use on Groupwise and Jabber, experimental Jingle voice chats on Jabber, offline Jabber messages, Jabber Transports, and AIM chatroom support (details <a href="http://kopete.kde.org/roadmap.php">here</a>). This version requires KDE 3.3 or later. Screenshots (last version 0.10) <a href="http://kopete.kde.org/screenshots.php">here</a>. Downloads (source, Debian Etch, Ubuntu Breezy &amp; Dapper, SUSE 32- and 64-bit RPMs, and Archlinux package) <a href="http://kopete.kde.org/releases.php">here</a>. The stable version is in the kdenetwork KDE package.</p>#
datePublished: #Mon Mar 13 12:13:59 2006#
dateUpdated: #Mon Mar 13 12:13:59 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Apps#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2670#
title: #Bibble v4.6 released#
link: #http://www.blogistan.co.uk/qt/2006/03/bibble_v46_released.php#
description: #From KDE Apps: Version 4.6 of Bibble, a Qt-based shareware RAW graphics app, has been released. The new features include &quot;Advanced Noise Reduction with Noise Ninja Technology&quot;, making this, according to the KDE Apps page&apos;s author, the only Linux software...#
content: #<p><a title="�Bibble KDE-Apps.org" href="http://www.kde-apps.org/content/show.php?content=36327">From KDE Apps</a>:</p> <p>Version 4.6 of <a href="http://www.bibblelabs.com/">Bibble</a>, a Qt-based shareware RAW graphics app, has been released. The new features include "Advanced Noise Reduction with Noise Ninja Technology", making this, according to the KDE Apps page's author, the only Linux software supporting sophisticated noise suppression; it also features faster Canon RAW conversion and a cleaner and more configurable UI interface. Single screenshot on the KDE Apps page. Shareware downloads <a href="http://www.bibblelabs.com/download.html">here</a>; the "lite" version costs $69, the pro version $129. Works on Windows, Mac and Linux.</p>#
datePublished: #Sat Mar 11 19:55:56 2006#
dateUpdated: #Sat Mar 11 19:55:56 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #Qt Applications#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2664#
title: #LinuxQuestions.org voters choose KDE as best desktop#
link: #http://www.blogistan.co.uk/qt/2006/03/linuxquestionsorg_voters_choos.php#
description: #From KDE Dot News: KDE won the LinuxQuestions.org &quot;Best Destktop Environment of the Year&quot; award by a convincing majority: 64.86% (1,253 votes) against GNOME&apos;s 25.67% and XFCE&apos;s 8.8%. Konqueror won for Best File Manager with 51.25% (Nautilus 15.93%), Quanta for...#
content: #<p><a title="KDE Wins Best Desktop Environment Award" href="http://dot.kde.org/1141951094/">From KDE Dot News</a>:</p> <p>KDE won the LinuxQuestions.org <a href="http://www.linuxquestions.org/questions/showthread.php?t=409028">"Best Destktop Environment of the Year"</a> award by a convincing majority: 64.86% (1,253 votes) against GNOME's 25.67% and XFCE's 8.8%. Konqueror won for <a href="http://www.linuxquestions.org/questions/showthread.php?t=409026">Best File Manager</a> with 51.25% (Nautilus 15.93%), Quanta for best web development editor (44.31%), and amaroK for best audio multimedia app (41.86%). KDevelop also came a very close second, after Eclipse, in the best IDE stakes. Summary <a href="http://www.linuxquestions.org/questions/showthread.php?t=422222">here</a>, full results <a href="http://www.linuxquestions.org/questions/forumdisplay.php?f=69">here</a>.</p>#
datePublished: #Fri Mar 10 12:03:19 2006#
dateUpdated: #Fri Mar 10 12:03:19 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE Community#
# Category end ####################
# Item end ########################
# Item begin ######################
id: #tag:www.blogistan.co.uk,2006:/qt//1.2663#
title: #Tectonic: KDE 4 preview#
link: #http://www.blogistan.co.uk/qt/2006/03/tectonic_kde_4_preview.php#
description: #From KDE Dot News: South African Free Software magazine Tectonic published a preview of KDE 4. Written by AJ Venter of South African distribution OpenLab the article describes how the KDE developers are going to &quot;rewrite the desktop rules with...#
content: #<p><a title="Tectonic: An Outlook on KDE 4" href="http://dot.kde.org/1141952466/">From KDE Dot News</a>:</p> <p>South African Free Software magazine <a href="http://www.tectonic.co.za">Tectonic</a> published a <a href="http://www.tectonic.co.za/TectonicJanFeb2006.pdf">preview of KDE 4</a>. Written by AJ Venter of South African distribution <a href="http://www.getopenlab.com">OpenLab</a> the article describes how the KDE developers are going to "<em>rewrite the desktop rules with KDE 4</em>" with technologies such as <a href="http://plasma.kde.org">Plasma</a>, <a href="http://solid.kde.org">Solid</a> and of course <a href="http://www.trolltech.com">Qt 4</a>. The article concludes that "<em>when KDE4 comes out, it will spell the end of the traditional way of using a computer</em>".</p>#
datePublished: #Fri Mar 10 11:52:18 2006#
dateUpdated: #Fri Mar 10 11:52:18 2006#
# Person begin ####################
name: #Indigo Jo#
uri: #http://www.blogistan.co.uk/blog/#
# Person end ######################
# Category begin ##################
term: #KDE#
# Category end ####################
# Category begin ##################
term: #Magazines#
# Category end ####################
# Item end ########################
# Feed end ########################

